interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -811,6 +811,17 @@ public AccessToken refreshAccessToken() throws IOException {
         authScope.contains(response.getOauthScope()));
   }
 
+  /** Sends an rpc to an unimplemented method on the server. **/
+  public void unimplementedMethod() {
+    UnimplementedServiceGrpc.UnimplementedServiceBlockingStub stub =
+        UnimplementedServiceGrpc.newBlockingStub(channel);
+    try {
+      stub.unimplementedCall(Empty.getDefaultInstance());
+    } catch (Throwable t) {
+      assertEquals(Status.UNIMPLEMENTED, Status.fromThrowable(t));
```

Comment:
Talked to @a11r, he would like to stick with no description if it does not add additional information.
Removed server side description.